client/my-sites/sidebar/style.scss

```diff
@@ -3,10 +3,14 @@
 	height: 24px;
 	position: absolute;
 	right: 16px;
-	top: 6px;
+	top: 8px;
 
 	@include breakpoint( "<660px" ) {
 		top: 13px;
 		right: 32px;
 	}
 }
+
+.sidebar__menu li.stats a:after {
+	background: none;
+}
```

Comment:
Nitpick: Should have end-of-file linebreak ([EditorConfig plugin](http://editorconfig.org/) helps for this).